parser/fuzz/fuzz_targets/statements.rs

```diff
@@ -0,0 +1,46 @@
+#![no_main]
+
+use ezno_parser::{ASTNode, Module, ParseOutput, SourceId, ToStringSettingsAndData};
+use libfuzzer_sys::{fuzz_target, Corpus};
+use pretty_assertions::assert_eq;
+use std::str;
+
+/// `do_fuzz` will take an arbitrary string, parse once and see if it returned a valid AST
+/// then it will print and parse that AST a second time and compare the printed outputs.
+/// If the second parse has a ParseError, that's a bug!!
+fn do_fuzz(data: &str) -> Corpus {
```

Comment:
function is broken out of `fuzz_target!()` to reduce work `rust-analyzer` has to do re: macros